### DISCUSSION TITLE
FDS.exe error

### DISCUSSION THREAD

**Discussion by PurnaPushkalaSomasundaram** [firemodels/fds#12779] [Q&A]
Hello. This is Purna and I have been facing problems with calling and running a.fds file using the command prompt. I am always called for this error notification as below. Does anyone know what is the problem about?
![WhatsApp Image 2024-04-12 at 14 01 10_b545d0e6](https://github.com/firemodels/fds/assets/166021559/a982bab9-40ce-4826-9b1c-b78621822fcb)

**Comment by drjfloyd** [comment]
How are you running FDS? Are you following the instructions in the User's Guide?

**Reply by PurnaPushkalaSomasundaram** [reply]
Yes, I am following the user guide.

**Reply by drjfloyd** [reply]
So you are using the CMDFDS shortcut and running using fds_local?

**Reply by PurnaPushkalaSomasundaram** [reply]
yes sir

**Reply by PurnaPushkalaSomasundaram** [reply]
fds_local casename.fds

**Reply by drjfloyd** [reply]
Try reinstalling the current release of FDS.

**Reply by PurnaPushkalaSomasundaram** [reply]
I found the error and thanks for the help.

**Comment by PurnaPushkalaSomasundaram** [comment]
I usually run in with pyrosim , This time I wanted to run with FDS software and found this error while running the files.
